### PR TITLE
Feature/tutorial

### PR DESCRIPTION
**Pull request by NotRaghavAggarwal**
Added tutorial functionality. Please approve <3 Thanks

### CODE REVIEW

**Review by Minirobbo — CHANGES_REQUESTED** [review]
For this to be pulled to main I think it would be good to see some actual dialogue added, possibly using the existing dialogue boxes so there is consistent UI throughout the game.

**Comment by connor-golin** [comment]
what does the tutorial currently do? 
I can see there's a new map and a text box in the bottom left, but please add more details to your pr description

also, pressing `F` stops player movement

File: source/core/assets/saves/savefile.json
Comment (Foref) [inline source/core/assets/saves/savefile.json:1]: Delete this file to resolve this specific conflict

Comment (NotRaghavAggarwal) [reply source/core/assets/saves/savefile.json:1]: removed

**Comment by rachitchaurasia** [comment]
Generated new pull request from tut2 branch that's already merged into main, so closing this one.

**Pull request closed without merging**